libr/core/cmd_cmp.c

```diff
@@ -304,59 +378,196 @@ static int radare_compare(RCore *core, const ut8 *f, const ut8 *d, int len, int
 	return len - eq;
 }
 
-static void cmd_cmp_watcher(RCore *core, const char *input) {
-	char *p, *q, *r = NULL;
-	int size = 0;
-	ut64 addr = 0;
+/* Returns 0 if operation succeeded, 1 otherwise */
+static int cmd_cmp_watcher(RCore *core, const char *input) {
+	static const char *help_msg_cw[] = {
+		"Usage: cw", "[args]", "Manage compare watchers; See if and how memory changes",
+		"cw??", "", "Show more info about watchers",
+		"cw ", "addr sz cmd", "Add a compare watcher",
+		"cw", "[*q] [addr]", "Show compare watchers (*=r2 commands, q=quiet)",
+		"cwd", " [addr]", "Delete watcher",
+		"cwr", " [addr]", "Revert watcher",
+		"cwu", " [addr]", "Update watcher",
+		NULL
+	};
+	static const char *verbose_help_cw =
+		"Watchers are used to record memory at 2 different points in time, then\n"
+		"report if and how it changed. First, create one with `cw addr sz cmd`. This\n"
+		"will record sz bytes at addr. To record the second state, use `cwu`. Now, when\n"
+		"you run `cw`, the watcher will report if the bytes changed and run the command given\n"
+		"at creation with the size and address. You may overwrite any watcher by creating\n"
+		"another at the same address. This will discard the existing watcher completely.\n"
+		"\n"
+		"When you create a watcher, the data read from memory is marked as \"new\". Updating\n"
```

Comment:
Use single quote ' instead of " so u dont have to escape it al the time

Reply:
Double quotes in the output look better imo